docs/docs/writing-your-own-enclave-host.md

```diff
@@ -307,20 +301,20 @@ if (Files.exists(enclaveStateFile)) {
 }
 ```
 
-The first command line parameter is a reference to the host directory which will contain the file for the enclave's 
-encrypted file system and also a file for the enclave's "sealed state". Both of these parameters are optional and 
-don't need to be specified if it's known the enclave will not use them.
+In this tutorial, you can pass the parameters of `EnclaveHost.start` method from the command line. The first 
+parameter is a reference to the host directory, which will contain the file for the enclave's encrypted file system 
```

Comment:
When the encrypted file system is mentioned reference `persistence.md#conclave-filesystems`. When the "sealed state" is mentioned reference `persistence.md#persistent-map`